editor/script/script_text_editor.cpp

```diff
@@ -2189,12 +2197,54 @@ bool ScriptTextEditor::can_drop_data_fw(const Point2 &p_point, const Variant &p_
 					String(d["type"]) == "nodes" ||
 					String(d["type"]) == "obj_property" ||
 					String(d["type"]) == "files_and_dirs")) {
+		set_drop_info_text(d);
 		return true;
 	}
 
 	return false;
 }
 
+void ScriptTextEditor::set_drop_info_text(const Dictionary &p_info) const {
+	String text;
+	String c = keycode_get_string((Key)KeyModifierMask::CMD_OR_CTRL);
+
+	static const String single_resource_text = TTR("Drag and drop to copy the file path into the script.") +
```

Comment:
```suggestion
	static const String single_resource_text = TTR("Drag and drop to copy the file path to the script.") +
```
I'd say to match the rest